owncloud/ios-app
# UI for navigating between folders in current path

## Description
Added an UI for fast and easy navigating in current path structure to parent folders.

## Related Issue
#354 

## Motivation and Context
Select a parent folder with one action, without navigating to all ViewControllers.

## How Has This Been Tested?
1. Open a deep nested folder structure
2. Tap on the navigation bar title (current folder name)
3. Select a folder in the popup view
4. Content of selected folder should be visible

## Screenshots (if appropriate):
![IMG_8088-2](https://user-images.githubusercontent.com/736109/55856055-74284280-5b69-11e9-872e-fe77a64e192e.png)

## Types of changes


## Checklist:
